docs/roadmap/README.md

```diff
@@ -25,17 +25,17 @@ gantt
     title Pocket V1 Roadmap
     dateFormat  YYYY-MM-DD
     section Milestone 1
-        Pocket PoS       :a1, 2022-06-01, 180d
+        Pocket PoS       :a1, 2022-06-01, 333d
     section Milestone 2
-        Pocket DoS       :a1, 2022-07-01, 150d
+        Pocket DoS       :a1, 2022-07-01, 303d
     section Milestone 3
-        Pocket RoS       :a1, 2022-12-01, 120d
+        Pocket RoS       :a1, 2022-12-01, 150d
     section Milestone 4
-        Pocket CoS       :a1, 2022-12-01, 120d
+        Pocket CoS       :a1, 2022-12-01, 150d
     section Milestone 5
-        Pocket IoS       :a1, 2023-04-01, 45d
+        Pocket IoS       :a1, 2023-05-01, 45d
     section Milestone 6
-        Pocket FoS       :a1, 2023-04-01, 90d
+        Pocket FoS       :a1, 2023-05-01, 90d
```

Comment:
I do not think we will have a mainnet launch end of July realistically speaking. It's beginning to look like a testnet next summer...